dist/PublicLab.Editor.js

```diff
@@ -370,7 +369,7 @@ function create (options) {
     bureaucrat.emit('valid', validFiles);
     var form = new FormData();
     Object.keys(o.formData).forEach(function copyFormData(key) {
-      form[key] = o.formData[key];
+      form.append(key, o.formData[key]);
```

Comment:
Ah, was this change from recent uncompiled changes or is it accidentally in dist? Thanks!